signac/contrib/project.py

```diff
@@ -629,29 +898,44 @@ def find_jobs(self, filter=None, doc_filter=None):
         statepoints, whereas the `doc_filter` argument compares against job
         document keys.
 
-        :param filter: A mapping of key-value pairs that all
-            indexed job statepoints are compared against.
-        :type filter: Mapping
-        :param doc_filter: A mapping of key-value pairs that all
-            indexed job documents are compared against.
-        :type doc_filter: Mapping
-        :yields: Instances of :class:`~signac.contrib.job.Job`
-        :raise TypeError: If the filters are not JSON serializable.
-        :raises ValueError: If the filters are invalid.
-        :raises RuntimeError: If the filters are not supported
-            by the index.
+        Parameters
+        ----------
+        filter : Mapping
+            A mapping of key-value pairs that all indexed job statepoints are
+            compared against (Default value = None).
+        doc_filter : Mapping
+            A mapping of key-value pairs that all indexed job documents are
+            compared against (Default value = None).
+
+        Returns
+        -------
+        signac.contrib.project.JobsCursor
+            JobsCursor of jobs matching the provided filter(s).
+
+        Raises
+        ------
+        TypeError:
```

Comment:
This is super nitpicky, but sometimes the errors have a colon after them and sometimes they don't. E.g., line 917 is `TypeError:` but line 873 is `TypeError`

Reply:
I think no colon is correct, based on the style guide here: https://numpydoc.readthedocs.io/en/latest/format.html
```suggestion
        TypeError
```

Reply:
I agree, I think I removed this elsewhere.

Reply:
corrected it.